### DISCUSSION TITLE
sorting order doesn't take typescript aliases into consideration

### DISCUSSION THREAD

**Discussion by luisrudge** [biomejs/biome#1312] [Ideas]
### Environment information

```bash
CLI:
  Version:                      1.4.1
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v18.18.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "npm/9.8.1"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 false

Workspace:
  Open Documents:
```

### Configuration

```JSON
{
  "$schema": "https://biomejs.dev/schemas/1.4.1/schema.json",
  "organizeImports": {
    "enabled": true
  }
}
```

### Playground link

https://biomejs.dev/playground/?lintRules=all&enabledLinting=false&code=aQBtAHAAbwByAHQAIABPAHAAZQBuAEEASQAgAGYAcgBvAG0AIAAnAG8AcABlAG4AYQBpACcAOwAKAGkAbQBwAG8AcgB0ACAAewAgAHoAIAB9ACAAZgByAG8AbQAgACcAegBvAGQAJwA7AAoAaQBtAHAAbwByAHQAIAAqACAAYQBzACAAbQBlAHQAcgBpAGMAcwAgAGYAcgBvAG0AIAAnAGIAYQBjAGsAZQBuAGQALwBsAGkAYgAvAG0AZQB0AHIAaQBjAHMAJwA7AAoAaQBtAHAAbwByAHQAIAB7ACAAZABlAGwAYQB5ACAAfQAgAGYAcgBvAG0AIAAnAGIAYQBjAGsAZQBuAGQALwB1AHQAaQBsAHMALwBzAHkAcwB0AGUAbQAnADsACgBpAG0AcABvAHIAdAAgAHMAdABhAHQAaQBjAFIAZQBzAHAAbwBuAHMAZQBzACAAZgByAG8AbQAgACcALgAvAGEAaQAuAHMAdABhAHQAaQBjAFIAZQBzAHAAbwBuAHMAZQBzACcAOwAKAGkAbQBwAG8AcgB0ACAAbABvAGcAZwBlAHIAIABmAHIAbwBtACAAJwBiAGEAYwBrAGUAbgBkAC8AbABpAGIALwBsAG8AZwBnAGUAcgAnADsA

### Screenshot
<img width="1375" alt="image" src="https://github.com/biomejs/biome/assets/941075/4258ff76-1d12-412a-a32a-da811c1c755a">

### Explanation
`backend` is an alias and it shouldn't be on the top of the list

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
This isn't something that Biome supports: https://biomejs.dev/analyzer/#how-imports-are-sorted

So I wouldn't consider this a bug, but a feature request. I suggest you to open a GitHub discussion and close this issue 😃

**Reply by marmotz** [reply]
Not a bug, but a needed feature !

Or the possibility of specifying an order of imports, as in @trivago/prettier-plugin-sort-imports https://github.com/trivago/prettier-plugin-sort-imports?tab=readme-ov-file#importorder

**Comment by elie222** [comment]
Would love to see progress on this. Anyone using the standard Next.js format will have lots of `@/component` type imports, but most would prefer to see imports like `react` at the top of the file instead. Would love to be able to tell biome to put `@/` imports at the bottom instead. You can use groups to do this, but that's manual work on every file you work on.

**Reply by Conaclos** [reply]
The following RFC should make you happy: #3015

**Comment by ematipico** [comment]
Closing in favour of https://github.com/biomejs/biome/discussions/3015
